### PR TITLE
Add OV layer sigmoid

### PR DESCRIPTION
**Pull request by cnamrata15**

### CODE REVIEW

File: plaidml/bridge/openvino/src/plaidml_plugin/ops/sigmoid.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/ Copyright(C) 2020 Intel Corporation
```
Comment (tzerrell) [inline plaidml/bridge/openvino/src/plaidml_plugin/ops/sigmoid.cpp:1]: Missing `/`

File: plaidml/bridge/openvino/src/plaidml_plugin/ops/sigmoid.cpp [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+#include "plaidml/op/op.h"
+
+    using namespace plaidml;      // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("sigmoid", [](const Context& ctx) {
+  auto* layer = dynamic_cast<ngraph::opset1::Sigmoid*>(ctx.layer);
```
Comment (tzerrell) [inline plaidml/bridge/openvino/src/plaidml_plugin/ops/sigmoid.cpp:18]: Don't include the creation of a `layer` variable if you never need to use it (e.g. on ops with no attributes). It causes compiler errors due to the unused variable.

**Comment by tzerrell** [comment]
Edited title for more context

File: plaidml/bridge/openvino/src/plaidml_plugin/ops/sigmoid.cpp (hunk 2 above)
Comment (cnamrata15) [reply plaidml/bridge/openvino/src/plaidml_plugin/ops/sigmoid.cpp:18]: I see. It usually gives a warning though.
I tried removing it, but the errors persist :(

Comment (tzerrell) [reply plaidml/bridge/openvino/src/plaidml_plugin/ops/sigmoid.cpp:18]: Yes, we have warnings as errors turned on, so the warning becomes an error.

Hmm, the same error persists? And you fixed the `/` -> `//` comment above?

**Pull request merged**